index.html

```diff
@@ -0,0 +1,265 @@
+                <h2 class="art">Profesional Art Printing Data</h2>
+                <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard</p>
+                <li class="listbutton2">
+                    <ul><button type="button" class="button6"> html</button></ul>
+                    <ul><button type="button" class="button7"> bootstrap</button></ul>
+                    <ul><button type="button" class="button8"> Ruby</button></ul>
+                </li>
+            </div>
+            </div>
+            <button type="button" class="button9"> See Project</button>
+            </div>
+            <!-- sixth card -->
+            <div class="listproject">
+                <h2 class="art">Profesional Art Printing Data</h2>
+                <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard</p>
+                <li class="listbutton2">
+                    <ul><button type="button" class="button6"> html</button></ul>
+                    <ul><button type="button" class="button7"> bootstrap</button></ul>
+                    <ul><button type="button" class="button8"> Ruby</button></ul>
+                </li>
+            </div>
+            </div>
+            <button type="button" class="button9"> See Project</button>
+            </div>
+
+            <section class="footersection">
+                <div class="aboutmefooter">
+                    <h2 class="footertext"> About me</h2>
+                    <p class="footer">Hello I’m a software developer! I can help you build a product , feature or
+                        website
+                        Look through some of my work and experience! If you like what you see and have a project you
+                        need coded, don’t hestiate to contact me.</p>
+                    <button type="button" class="button10"> Get My Resume</button>
+                </div>
+                <footer class="languagesection">
+                    <div class="languageskill1">
+                        <h2 class="language"><img src="./images/Rectangle 55.svg" alt="Rectangle" class="rectangle" />
+                            Language</h2>
+                        <li class="languagelist">
+                            <ul class="item1">
+                                <p>Javascript</p>
+                            </ul>
+                            <ul class="item2">
+                                <p>Ruby</p>
+                            </ul>
+                            <ul class="item3">
+                                <p>Html</p>
+                            </ul>
+                            <ul class="item4">
+                                <p>Css</p>
+                            </ul>
+                        </li>
```

Comment:
- [ ] Kindly make sure you use semantic tag following the Web3C standard.
`It's supposed to be:`
`<ul>`
    `<li>JavaScript</li>`
     `<li>Ruby</li>`
`</ul>`
`And not the other way around. Please consider changing that.`